examples/c/GNUmakefile

```diff
@@ -0,0 +1,39 @@
+# Copyright 2016-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
It might be helpful to add a brief README to each of these examples to help users configure these builds.

Reply:
done.